babel.config.js

```diff
@@ -7,5 +7,4 @@ module.exports = {
     '@babel/preset-env',
     '@babel/preset-react'
   ],
-  plugins: ['@babel/plugin-proposal-class-properties']
```

Comment:
This needs to go back in with the revert.

Reply:
My bad. I've revert the wrong commit